python/lsst/ts/Dome/dome_csc.py

```diff
@@ -62,8 +60,8 @@ def __init__(
         self.lower_level_status = None
         self.status_task = None
 
-        self.amcl = AmcsConfigurationLimits()
-        self.lwscl = LwscsConfigurationLimits()
+        self.amcs_limits = AmcsLimits()
```

Comment:
The mock_port needs to be specified when using simulation mode. No random port will be assigned in case of simulation mode and mock_port = None. Setting the port via the command line is not supported. In case of non-simulation mode the port is taken from the configuration.

Reply:
Done as discussed on Slack.